test/evergreen.yml

```diff
@@ -5584,7 +5584,7 @@ buildvariants:
       WT_BUILDDIR=$WT_TOPDIR/cmake_build
       ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0:unmap_shadow_on_exit=1"
       LSAN_OPTIONS="print_suppressions=0:suppressions=$WT_TOPDIR/test/evergreen/asan_leaks.supp"
-      ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer
+      ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v4/bin/llvm-symbolizer
       TESTUTIL_BYPASS_ASAN=1
       LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libeatmydata.so:$WT_TOPDIR/TCMALLOC_LIB/lib/libtcmalloc.so
```

Comment:
`libeatmydata.so` is the only other library we're accessing from `/usr/lib` instead of via the mongotoolchain. This might be worth a spinoff ticket.

CC @lukech as I'm not sure how we decide whether dependencies are added to the mongo toolchain.

Reply:
We had [BUILD-9065](https://jira.mongodb.org/browse/BUILD-9065) that added the eatmydata package onto the Ubuntu & RHEL distro in the past. I guess we (WiredTiger) are the only team that makes use of the package in testing, and not sure if it's a good candidate to be added to the mongo toolchain.